src/libraries/Common/src/Interop/Windows/Ole32/Interop.CoGetObjectContext.cs

```diff
@@ -3,13 +3,15 @@
 
 using System;
 using System.Runtime.InteropServices;
-using System.Runtime.InteropServices.ComTypes;
 
 internal static partial class Interop
 {
     internal static partial class Ole32
     {
+#pragma warning disable DLLIMPORTGENANALYZER015 // Use 'GeneratedDllImportAttribute' instead of 'DllImportAttribute' to generate P/Invoke marshalling code at compile time
+        // TODO: [DllImportGenerator] Switch to use GeneratedDllImport once we annotate blittable types used in interop in CoreLib (like Guid)
         [DllImport(Libraries.Ole32)]
         internal static extern int CoGetObjectContext([MarshalAs(UnmanagedType.LPStruct)] Guid riid, out IntPtr ppv);
```

Comment:
Since this is COM, if it isn't too big of a change I suppose we could change this to `Guid*`?

Reply:
We have a bunch of similar things around Guid, so I figure we can just keep their existing signatures and do a pass when  we enable it (it is already noted in https://github.com/dotnet/runtime/issues/60595).